Corona-Warn-App/src/test/java/de/rki/coronawarnapp/presencetracing/checkins/checkout/CheckOutHandlerTest.kt

```diff
@@ -68,7 +83,37 @@ class CheckOutHandlerTest : BaseTest() {
             checkInEnd = nowUTC,
             completed = true
         )
-        // TODO journal creation
+
+        coVerify(exactly = 1) {
+            contactJournalCheckInEntryCreator.createEntry(any())
+        }
+
         // TODO cancel auto checkouts
     }
+
+    @Test
+    fun `Creates entry if create journal entry is true`() = runBlockingTest {
+        createInstance().apply {
+            checkOut(42)
+        }
+
+        updatedCheckIn?.createJournalEntry shouldBe true
```

Comment:
No `rofl?.copter shouldBe XYZ` in tests please. If it should not be null, force a check `!!`.

Reply:
In this case it won't make a difference. If it's null the test will fail anyways